packages/trader/src/Modules/CFD/Containers/cfd-dashboard.jsx

```diff
@@ -208,367 +204,365 @@ class CFDDashboard extends React.Component {
                                     )}
-                                </div> */}
-                                <div className='cfd-dashboard__download-center'>
-                                    <h1 className='cfd-dashboard__download-center--heading'>
-                                        {platform === CFD_PLATFORMS.MT5 && (
-                                            <Localize i18n_default_text='Download the MT5 app' />
-                                        )}
-                                        {platform === CFD_PLATFORMS.DXTRADE && (
-                                            <Localize i18n_default_text='Download the Deriv X app' />
-                                        )}
-                                    </h1>
-                                    <div className='cfd-dashboard__download-center-options--mobile'>
-                                        <div className='cfd-dashboard__download-center-options--mobile-devices'>
-                                            {platform === CFD_PLATFORMS.MT5 && (
-                                                <React.Fragment>
-                                                    <Icon icon='IcMt5DeviceTablet' width={133} height={106} />
-                                                    <Icon icon='IcMt5DevicePhone' width={48} height={74} />
-                                                </React.Fragment>
-                                            )}
-                                            {platform === CFD_PLATFORMS.DXTRADE && (
-                                                <React.Fragment>
-                                                    <Icon icon='IcDxtradeDeviceTablet' width={133} height={106} />
-                                                    <Icon icon='IcDxtradeDevicePhone' width={48} height={74} />
-                                                </React.Fragment>
-                                            )}
+                                    {platform === CFD_PLATFORMS.MT5 && (
+                                        <Localize i18n_default_text='Server maintenance starting 01:00 GMT every Sunday. This process may take up to 2 hours to complete. Service may be disrupted during this time.' />
+                                    )}
+                                </div>
+                            </div>
+                        </div>
+                        <DesktopWrapper>
+                            <CFDDashboardContainer
+                                platform={platform}
+                                active_index={active_index}
+                                is_dark_mode_on={is_dark_mode_on}
+                            />
+                        </DesktopWrapper>
+                        <MobileWrapper>
+                            {/* <div className='cfd-dashboard__download-center'>
+                                If you have the app, launch it
```

Comment:
Let's remove all the commented code in this file. If needed, we can always access them using git!